src/styles/shards.less

```diff
@@ -10,6 +10,12 @@
     .progress-bar {
         z-index: 1;
     }
+
+    .collapsing {
+        -webkit-transition: height 0s;
+        transition: height 0s;
```

Comment:
Any particular reason to specify the `-webkit-` browser prefix only? It looks like all major browsers have supported the unprefixed version since ~2013.

Speaking of browser prefixes, I learnt recently that [autoprefixer](https://github.com/postcss/autoprefixer) is a thing that exists. Might be useful when we eventually try to make the game work nicely with older browser versions.

Reply:
That looks pretty good, although not sure how far back we want to go in supported versions.
But at least if that was implemented it's one less thing to worry about.